internal/connection_repository/types.go

```diff
@@ -28,9 +28,8 @@ type ConnectionRegistrar interface {
 
 type ConnectionLocator interface {
 	GetConnection(context.Context, domain.AccountID, domain.OrgID, domain.ClientID) controller.ConnectorClient
-	GetConnectionsByAccount(context.Context, domain.AccountID, int, int) (map[domain.ClientID]controller.ConnectorClient, int, error)
-	GetAllConnections(context.Context, int, int) (map[domain.AccountID]map[domain.ClientID]controller.ConnectorClient, int, error)
 }
 
 type GetConnectionByClientID func(context.Context, *logrus.Entry, domain.OrgID, domain.ClientID) (domain.ConnectorClientState, error)
 type GetConnectionsByOrgID func(context.Context, *logrus.Entry, domain.OrgID, int, int) (map[domain.ClientID]domain.ConnectorClientState, int, error)
+type GetAllConnections func(context.Context, int, int) (map[domain.AccountID]map[domain.ClientID]controller.ConnectorClient, int, error)
```

Comment:
Let's return this to return the ConnectorClientState struct instead of a proxy:
```suggestion
type GetAllConnections func(context.Context, int, int) (map[domain.AccountID]map[domain.ClientID]domain.ConnectorClientState, int, error)
```